pkg/util/admission/granter.go

```diff
@@ -1976,12 +2229,14 @@ func (res adjustTokensResult) SafeFormat(p redact.SafePrinter, _ rune) {
 	p.Printf("%s unacc [≈%s/req, n=%d], ",
 		ib(res.aux.intUnaccountedL0Bytes), ib(int64(res.smoothedIntPerWorkUnaccountedL0Bytes)), res.aux.intAdmittedCount)
 	// How much got compacted out of L0 recently.
-	p.Printf("compacted %s [≈%s]; ", ib(res.aux.intL0CompactedBytes), ib(res.smoothedIntL0CompactedBytes))
-
+	p.Printf("compacted %s [≈%s], ", ib(res.aux.intL0CompactedBytes), ib(res.smoothedIntL0CompactedBytes))
+	p.Printf("flush tokens %s with multiplier %.3f [≈%s]; ", ib(int64(res.aux.intFlushTokens)),
```

Comment:
This isn't the right place for the multiplier, no? I would expect something here like

```go
p.Printf("flushed %s [≈%s]; ", ib(int64(res.aux.intFlushTokens)), ib(res.smoothedIntFlushTokens))
```

so basically just printing the flush stats.

Then, in the "admitting" section, it should say whether the token count was derived from L0 or from flushes:

> admitting 4.02mb/s due to L0 growth with L0 penalty: X, Y, flush multiplier %.3f
> admitting 4.02mb/s due to memtable flush with L0 penalty: X, Y, flush multiplier %.3f